release/models/acl/openconfig-packet-match.yang

```diff
@@ -329,6 +341,18 @@ module openconfig-packet-match {
         "Destination IPv4 address prefix.";
     }
 
+    leaf masked-destination-address {
+      type oc-inet:ipv4-address;
+      description
+        "Destination IPv4 address used with a mask.";
+    }
+
+    leaf destination-address-mask {
+      type oc-inet:ipv4-address;
+      description
+        "Destination IPv4 address mask.";
```

Comment:
Please specify that this is a wildcard mask. Maybe even better to add `wildcard` to the name of the leaf.